doc/platform.txt

```diff
@@ -42,6 +45,8 @@ Syntax
     <url> ............................. url to a git repository containing a platform
 
     <version> ......................... major.minor.patch version specifier
+
+    <branch> .......................... name of the branch to checkout after cloning the git repo
```

Comment:
If `#branch` is not specified after repo url, then `master` will be used by default, right? Could you please note this as well.